media-libs/partio/patches/partio-1.1.0.patchset

```diff
@@ -18,5 +21,32 @@ index e331c9c..9c592ca 100644
  ## Search for useful libraries
  find_package(GLUT REQUIRED)
 -- 
-2.15.1
+2.24.1
+
+From 32bdb4e2973eba6f35c4982a94f87ec1a2a59bc7 Mon Sep 17 00:00:00 2001
+From: X512 <danger_mail@list.ru>
+Date: Fri, 31 Jan 2020 03:36:52 +0900
+Subject: CharArrayLen: fix build
+
+---
+ src/lib/io/MC.cpp | 4 ++--
+ 1 file changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/src/lib/io/MC.cpp b/src/lib/io/MC.cpp
+index 7125954..7460c0b 100644
+--- a/src/lib/io/MC.cpp
++++ b/src/lib/io/MC.cpp
+@@ -105,8 +105,8 @@ bool ReadAttrHeader(std::istream& input, Attribute_Header& attribute){
+ 
+ int CharArrayLen(char** charArray){
```

Comment:
Upstream has https://github.com/wdas/partio/blob/032a1ec24246b28ad26dbc7f163417c1b921de42/src/lib/io/MC.cpp#L106
Could this work (doesn't bail or errors out here)?